Lib/test/test_venv.py

```diff
@@ -24,8 +24,10 @@
 except ImportError:
     ctypes = None
 
-skipInVenv = unittest.skipIf(sys.prefix != sys.base_prefix,
-                             'Test not appropriate in a venv')
+requireVenvCreate = unittest.skipUnless(
+    sys.platform.startswith('win')
+    or sys.prefix == sys.base_prefix,
+    'Test requires venv.create')
```

Comment:
I don't understand why you changed the decorator name and the skip message. I don't understand the new skip message (see my other comment on the PR). Why not just add sys.platform.startswith('win') in skipInVenv?